physionet-django/project/templates/project/edit_files_panel.html

```diff
@@ -1,7 +1,20 @@
 {% load static %}
 
+{% block local_js_top %}
+<script src="{% static 'dropzone/dropzone.min.js' %}"></script>
+{% endblock %}
+
+{% block local_css %}
+<link rel="stylesheet" type="text/css" href="{% static 'dropzone/dropzone.min.css' %}">
```

Comment:
What's the difference between `dropzone.min.css` and `basic.min.css`? Do we need both?

Reply:
It seems like having `dropzone.min.css` is sufficient.